EveryonesInvited/website
# Form draft

### What changes have you made?

Created form template for testimonial submission 

### Which issue(s) does this PR fix?

Fixes #5 

### Screenshots (if there are design changes)

![Screenshot 2021-05-07 at 19 24 33](https://user-images.githubusercontent.com/65901987/117493030-560cb680-af6a-11eb-8a3a-e32f47fb5864.png)

### How to test

- `yarn start`
- go to localhost 3000
- complete form
- hit submit 
- check dev tools to see if form submission is logged.
